### PR TITLE
added pomodoro timer

### PR DESCRIPTION
**Pull request by GeetaPogul**
- added pomodoro timer
updated some changes and code

### CODE REVIEW

File: src/components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+.btns {
+  border: none;
+  padding: 1rem 2.3rem;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+.modal-add-btn {
+  background-color: var(--primary-color);
+  color: white;
```
Comment (xii-vi) [inline src/components/Modal/modal.css:69]: have consistency in naming

File: src/components/Navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@import url("../../root.css");
+html {
+  font-size: 10px;
+}
+body {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+nav {
+  background-color: var(--primary-color);
+  padding: 2rem 3rem 2rem 2rem;
```
Comment (xii-vi) [inline src/components/Navbar/navbar.css:10]: if you are using your own CL, then you must have utility classes for spacing.
_Try to use that._

File: src/components/PomoDesc/PomoDesc.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
```
Comment (xii-vi) [inline src/components/PomoDesc/PomoDesc.jsx:1]: you don't need to import react in every component

File: src/root.css [hunk 4]
```diff
@@ -0,0 +1,5 @@
+:root{
+    /* --primary-color:#208899; */
```
Comment (xii-vi) [inline src/root.css:2]: remove commented code

**Review by xii-vi — APPROVED** [review]
_Added a Few Comments.
GO through once before merging.
Also, try to include a deploy preview._

File: src/components/Footer/Footer.jsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+import "../Footer/footer.css";
+
+const Footer = () => {
+  return (
+    <>
+      <div className="m-top"> </div>
+
+      <div className="footer-container">
+        <div className="footer-icons">
+          <a className="footer-links" href="https://twitter.com/Geeta2005">
```
Comment (xsaurabhsuthar) [inline src/components/Footer/Footer.jsx:11]: suggestion: you should avoid using a tags in react.

File: src/components/PomoDesc/PomoDesc.jsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+import "./pomoDesc.css";
+const PomoDesc = () => {
+  const { description } = useModal();
+
+  return (
+    <>
+      <div className="pomodoro">
+        <div className="pomodoro-container">
+          {description.map((pomo) => {
+            return (
```
Comment (xsaurabhsuthar) [inline src/components/PomoDesc/PomoDesc.jsx:12]: suggestion: you can simply return it without using return statement.

File: src/pages/Landing/Landingpage.jsx [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+          <img className="landing-img" src="./home-img.png" alt="time-img" />
+        </div>
+
+        <div className="content-container">
+          <span>
+            People to work with the time they have rather then against it. Using
+            POMODORO, you break your workday into 25-minute separated by
+            five-minute breaks. These intervals are referred by pomodoros.
+            Basically is is a time management system.
+            <br /> <br />
```
Comment (xsaurabhsuthar) [inline src/pages/Landing/Landingpage.jsx:20]: avoid using such semantic html `<br/>`.

**Review by xsaurabhsuthar — APPROVED** [review]
Clean code. Good to merge 🚀🚀🚀🚀.

File: src/components/PomoDesc/PomoDesc.jsx (hunk 6 above)
Comment (tanishk15sharma) [reply src/components/PomoDesc/PomoDesc.jsx:12]: yes

**Review by tanishk15sharma — APPROVED** [review]
after few changes you can merge this PR , comments are listed above

**Pull request merged**